app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueFields/linkIssueFields.jsx

```diff
@@ -79,7 +83,7 @@ export class LinkIssueFields extends Component {
               name={`${issue}.issueLink`}
               fieldWrapperClassName={cx('field-wrapper')}
               label={this.props.intl.formatMessage(messages.issueLinkLabel)}
-              parse={this.parseValue}
+              onChange={this.handleIssueLinkChange}
```

Comment:
May be better: `onChange={withAutocomplete ? this.handleIssueLinkChange: null}`?

Reply:
There's almost no difference in this case, but with handler renaming it will look more clear, I think. Fixed.